Change block usersetting (#11195)
github/docs: data/reusables/organizations/moderation-settings.md

> 1. In your organization's Settings sidebar, click **Moderation settings**.

Reviewer: ```suggestion
1. In the left sidebar, click **Moderation settings**.
```
